CONTRIBUTING.md

```diff
@@ -107,6 +105,13 @@ git push origin bugfix-branch:bugfix-branch
 
 Go to the web page of GitHub fork repo, there would be a chance to create a Pull Request after pushing to a new branch, just click button "Compare & pull request" to do it. Then edit the description for proposed changes, which can just be copied from the commit message.
 
+Please sign the HugeGraph CLA when contributing code for the first time. You can sign the CLA by just posting a Pull Request Comment same as the below format:
+
+`I have read the CLA Document and I hereby sign the CLA`
+
+Note that please make sure the email address you submit the code is bound to the GitHub account. For how to bind the email address, please refer to https://github.com/settings/emails:
```

Comment:
typo and simplify: 

**Note:**  please make sure the email address you submit `with` the code `has` bound`ed` with GitHub account

Reply:
Here, most users use emails bound to the github, but some users have already submitted the code with different email and pushed it to github, then trigger this problem, so here is a note that more emails can be bound to github.